contracts/Oracle/SovrynSwapPriceFeed.sol

```diff
@@ -1,12 +1,13 @@
 // SPDX-License-Identifier: MIT
```

Comment:
We are already using SovrynSwapPriceFeed today, how those changes are gonna affect it? will we upgrade it to this updated and upgraded contract?